fairmq/tools/runSimpleMQStateMachine.h

```diff
@@ -30,6 +32,15 @@ inline int runStateMachine(TMQDevice& device, FairMQProgOptions& cfg)
         return 0;
     }
 
+    if (cfg.Count("version"))
+    {
+        std::cout << "User device version: " << device.GetVersion() << std::endl;
```

Comment:
If we do not print the FairRoot version here, we might call this just `device version`, what do you think?